webextensions/manifest/optional_permissions.json

```
@@ -831,6 +831,26 @@
             }
           }
         },
+        "userScripts": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "115"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136",
+                "notes": "Provided as an [optional-only permission](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/optional_permissions#optional-only_permissions)."
+              },
+              "firefox_android": "mirror",
```

The permission is not supported on Android either, until https://bugzilla.mozilla.org/show_bug.cgi?id=1949955 is fixed.
```suggestion
              "firefox_android": {
                "version_added": false
              },
```